pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -534,14 +534,6 @@ def _initialize(self):
         kickstart_timezone = self._timezone_module.Timezone
         if is_valid_timezone(kickstart_timezone):
             self._set_timezone(kickstart_timezone)
-        elif not flags.flags.automatedInstall:
-            log.warning("%s is not a valid timezone, falling back to default (%s)",
-                        kickstart_timezone, DEFAULT_TZ)
-            self._set_timezone(DEFAULT_TZ)
-            self._timezone_module.SetTimezoneWithPriority(
-                DEFAULT_TZ,
-                constants.TIMEZONE_PRIORITY_KICKSTART
-            )
```

Comment:
Could you please explain me why the correction from kickstart is not needed anymore? Is that because it is already set from the module?

Reply:
Yes.